configs/example/gem5_library/checkpoints/simpoints-se-checkpoint.py

```diff
@@ -110,33 +110,43 @@
     cache_hierarchy=cache_hierarchy,
 )
 
-# board.set_workload(
-#    Workload("x86-print-this-15000-with-simpoints")
-#
-# **Note: This has been removed until we update the resources.json file to
-# encapsulate the new Simpoint format.
 # Below we set the simpount manually.
-
-board.set_se_simpoint_workload(
-    binary=obtain_resource("x86-print-this"),
-    arguments=["print this", 15000],
-    simpoint=SimpointResource(
-        simpoint_interval=1000000,
-        simpoint_list=[2, 3, 4, 15],
-        weight_list=[0.1, 0.2, 0.4, 0.3],
-        warmup_interval=1000000,
-    ),
+# board.set_se_simpoint_workload(
+#     binary=obtain_resource("x86-print-this"),
+#     arguments=["print this", 15000],
+#     simpoint=SimpointResource(
+#         simpoint_interval=1000000,
+#         simpoint_list=[2, 3, 4, 15],
+#         weight_list=[0.1, 0.2, 0.4, 0.3],
+#         warmup_interval=1000000,
+#     ),
+# )
+
+# The workload x86-print-this-15000-with-simpoints, version 2.0.0 is available
+# for gem5 versions 23.1 through 24.1. We may need to create another version
+# of the workload that throws a hypercall (see comment below)
```

Comment:
Make an issue, please. If you want to go ahead and merge this PR, reference the issue here.